subsys/bluetooth/host/hci_core.c

```diff
@@ -4280,29 +4348,6 @@ int bt_disable(void)
 	/* Clear BT_DEV_READY before disabling HCI link */
 	atomic_clear_bit(bt_dev.flags, BT_DEV_READY);
 
-	err = bt_dev.drv->close();
```

Comment:
Do you remember the reason for reordering statements in `bt_disable()`?

Reply:
> Do you remember the reason for reordering statements in `bt_disable()`?

Yes. There were actions in `bt_disable()` that triggered HCI activity, so calling `close()` had to be moved after those.

Reply:
I believe we need https://github.com/zephyrproject-rtos/zephyr/pull/76203